src/Elastic.Markdown/Assets/pages-nav.ts

```diff
@@ -16,11 +16,14 @@ function scrollCurrentNaviItemIntoView(nav: HTMLElement, delay: number) {
 	expandAllParents(currentNavItem);
 	setTimeout(() => {
 		if (currentNavItem && !isElementInViewport(nav, currentNavItem)) {
-			currentNavItem.scrollIntoView({ behavior: 'smooth', block: 'center' });
-			window.scrollTo(0, 0);
+			const navRect = nav.getBoundingClientRect();
+			const currentNavItemRect = currentNavItem.getBoundingClientRect();
+			const offset = currentNavItemRect.top - navRect.top - navRect.height / 3 + currentNavItemRect.height / 2;
```

Comment:
Can you leave a comment documenting what the devisions accomplish here?

Reply:
https://github.com/elastic/docs-builder/pull/702/commits/648fd2eb6f1ff2565d500f4562a773282e9f9e16